src/offspot_demo/maint-compose/docker-compose.tmpl.yml

```diff
@@ -0,0 +1,6 @@
+services:
```

Comment:
Can we either make this useable directly for dev or move it to the setup.py?

Reply:
It's already part of the setup.py, I don't get it, sorry

Reply:
I mean it doesn't need to be in a separate file it it's that small and not usable directly ; can be a (multiline) string in the script